core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -111,6 +111,7 @@ private val QUALITY_RANGE_MAP = mapOf(
     HD to Range.create(720, 1079),
     SD to Range.create(241, 719)
 )
+private var zoomScaleMap = mutableMapOf<LensFacing, Float>()
```

Comment:
This is essentially a global variable, which we don't want to do. This should be declared within the camera coroutine in CameraXCameraUseCase and added to the CameraSessionContext.